tests/executors/test_executor_loader.py

```diff
@@ -36,23 +35,17 @@ class FakeExecutor:
 
 
 class TestExecutorLoader:
-    def setup_method(self) -> None:
-        from airflow.executors import executor_loader
+    @pytest.fixture(autouse=True)
+    def setup_method(self, clean_executor_loader) -> None:
```

Comment:
Should we merge `setup_method` and `teardown_method` as a fixture and make it `autouse` of `TestExecutorLoader`